src/components/selects-request/detail-drawer-content/index.tsx

```diff
@@ -64,7 +64,7 @@ function SelectsRequestDetailDrawerContent({
 
   return (
     <Wrapper>
-      <DrawerApplicantSection {...user} snsLink={snsLink} />
+      <DrawerApplicantSection {...user} snsLink={snsLink} uploaderType={user.uploaderType}/>
```

Comment:
```suggestion
      <DrawerApplicantSection {...user} snsLink={snsLink} />
```

 JSX Spread Attributes 읽어보세요! [링크](https://gist.github.com/sebmarkbage/07bbe37bc42b6d4aef81)

Reply:
4fcd357